src/components/StrategyLiveChart/StrategyLiveChart.js

```diff
@@ -74,16 +89,20 @@ const StrategyLiveChart = ({
       fullscreen={fullscreenChart}
       onExitFullscreen={exitFullscreenChart}
     >
+      {/* We have to use key prop for correct Chart rerender when strategy changed
+          https://medium.com/@albertogasparin/forcing-state-reset-on-a-react-component-by-using-the-key-prop-14b36cd7448e
+      */}
       {chartMarket && (
         <Chart
           market={chartMarket}
           theme={settingsTheme}
-          layoutI={`strategy-editor-${id}`}
+          layoutI={`strategy-editor-${executionId}`}
           indicators={chartIndicators}
           interval={interval}
           trades={trades}
           hideResolutions
           chartRange={chartRange}
+          key={executionId}
```

Comment:
I think this change is not required because `layoutI` is now correct with executionId, and with these prop change component will re-render already. but still let's keep it for now.

Reply:
@bhoomij no, it is not. `executionId` doesn't re-render forced